app/views/categories/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<% if user_signed_in? %>
+  
+     <%end%>
+<div class="container">
+        <%# <div class="card shadow-2-strong" style="border-radius: 1rem;"> %>
+
+<div class="header-contents">
+<h1 class="header">Categories</h1>
+<%= link_to "Add New Category", new_category_path, class: "btn btn-primary add-button"%>
+</div>
+
+<div id="categories">
+<table class="table">
+<thead class="thead-dark">
+  <tr>
+    <th scope="col">Icons</th>
+    <th scope="col">Category</th>
+    <th scope="col">Actions</th>
+  </tr>
+</thead>
+<tbody>
+<% @categories.each do |category| %>
+  <%= render category %>
+<% end %>
+
+</tbody>
```

Comment:
- [ ] Same here, As per this [requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=You%20should%20follow%20these%20design%20guidelines%2C%20including%3A), Please follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:

- Colors.
- Typography: font face, size and weight.
- Layout: composition and space between elements.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/208917236-b01d7d9a-e4a1-4197-ab8f-3206506571ad.png) | ![image](https://user-images.githubusercontent.com/68796232/208917196-3d1cac49-0da5-41fc-b2b8-aedd8dc18652.png) |